### DISCUSSION TITLE
Happy Holiday Season and a Spectacular New Year!

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#3786] [General]
Wish everyone a peaceful coming new year and a spectacular Christmas! I hope you all will be able to be with friends and family! 🤗 🤗 🤗 🤗

**Comment by Betree** [comment]
You too @SudharakaP! Hope 2021 will be calmer than 2020 :smile: Thank you for all your contributions this year, you were of great help.

And since this is a public discussion, thanks to everyone who contributed in 2020 whether it's code, documentation, design, ideas or reporting issues. You contributions matter, and we are glad to have you in our community. 

:heart: @snitin315 @bolariin @shftrg @IkeChukwuanu @jeffin143 @brymut @JoyLubega @leonardohofling @sarathsomana @vickywane @jamesgeorge007 @twisstosin @samcaspus @saideepesh000 @patcon @itsaaryan @combinatorist @chenxsan @bkardell @adong @MlkMahmud @InkFaust @CanerSezgin @RichardLitt @rista404 

(sorry if I forgot anyone!)

**Comment by jeffin143** [comment]
Merry Christmas and happy new year to everyone managing the project @Betree @znarf @alanna @sbinlondon  and to all others too :)

**Comment by patcon** [comment]
Y'all are wonderful. This project makes me hopeful :)
